backend/src/main/java/codesquad/airbnb/accomodation/AccommodationController.java

```diff
@@ -1,21 +1,46 @@
 package codesquad.airbnb.accomodation;
 
+import codesquad.airbnb.accomodation.dto.AccommodationSearchRequest;
 import codesquad.airbnb.accomodation.dto.PriceAndCountStatistics;
+import codesquad.airbnb.accomodation.dto.WishedAccommodationResponse;
 import codesquad.airbnb.accomodation.service.AccommodationStatisticsService;
+import codesquad.airbnb.accomodation.service.WishedAccommodationService;
 import lombok.RequiredArgsConstructor;
 import org.springframework.web.bind.annotation.GetMapping;
 import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RequestParam;
 import org.springframework.web.bind.annotation.RestController;
 
+import java.time.LocalDate;
+import java.time.format.DateTimeFormatter;
+import java.util.List;
+
 @RestController
 @RequiredArgsConstructor
-@RequestMapping("/accommodation")
+@RequestMapping("/api/accommodations")
 public class AccommodationController {
 
+    private static final DateTimeFormatter DATE_FORMAT = DateTimeFormatter.ofPattern("yyyy-MM-dd");
+
     private final AccommodationStatisticsService dummyAccommodationStatisticsService;
+    private final WishedAccommodationService wishedAccommodationService;
 
     @GetMapping("/statistics/price")
     public PriceAndCountStatistics getAccommodationPriceStatistics() {
         return dummyAccommodationStatisticsService.getCachedPriceAndCountStatistics();
     }
+
+    @GetMapping()
+    public List<WishedAccommodationResponse> getNotReservedAccommodations(
```

Comment:
필수는 아니고 참고사항 입니다.
not~~같은 부정문 대신 긍정문을 사용하면 좀 더 간결하고 명확한 경우가 있습니다 :)

```suggestion
    public List<WishedAccommodationResponse> getAvailableAccommodations(
```